src/source-map.ts

```diff
@@ -0,0 +1,192 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as acorn from 'acorn';
```

Comment:
Is it feasible to use espree instead? We already ship espree with the bundler, and it locks to a specific version of acorn, so I'm concerned that this will result in us shipping two copies of acorn.

Espree exposes its tokenizer I believe